inlong-sort/sort-connectors/base/src/main/java/org/apache/inlong/sort/base/metric/SourceMetricData.java

```diff
@@ -247,19 +229,4 @@ public void outputMetrics(long rowCountSize, long rowDataSize) {
                     rowDataSize);
         }
     }
-
-    @Override
-    public String toString() {
-        return "SourceMetricData{"
-                + "groupId='" + groupId + '\''
-                + ", streamId='" + streamId + '\''
-                + ", nodeId='" + nodeId + '\''
-                + ", numRecordsIn=" + numRecordsIn.getCount()
-                + ", numBytesIn=" + numBytesIn.getCount()
-                + ", numRecordsInForMeter=" + numRecordsInForMeter.getCount()
-                + ", numBytesInForMeter=" + numBytesInForMeter.getCount()
-                + ", numRecordsInPerSecond=" + numRecordsInPerSecond.getRate()
-                + ", numBytesInPerSecond=" + numBytesInPerSecond.getRate()
-                + '}';
-    }
```

Comment:
Add toString() for printing some data when invoke snapshotState. Because it is convenient to troubleshooting metrics problem.